src/app.dump.ts

```diff
@@ -14,7 +14,8 @@ interface Parameters {
 }
 
 export default async function dump({ env }: Parameters) {
-  const shell = flatmap(Deno.env.get("SHELL"), basename)
+  const { SHELL, TEA_REWIND } = useEnv();
+  const shell = flatmap(SHELL, basename)
```

Comment:
So my thought was to do all caps because they were env vars (and it was easier to change the spots where we used `Deno.env.get`) but yeah maybe that wasn't a great idea.  I could go either way.

Reply:
I as well. The undecorated ALL-CAPS vars look weird, but if you know why they're all-caps, it's communicative. @mxcl you have a horse in this race?